src/pheval/utils/file_utils.py

```diff
@@ -34,3 +38,41 @@ def obtain_closest_file_name(file_to_be_queried: Path, file_paths: list[Path]) -
     return [
         file_path for file_path in file_paths if Path(closest_file_match) == Path(file_path.name)
     ][0]
+
+
+def ensure_file_exists(*files: str):
+    """Ensures the existence of files passed as parameter
+    Raises:
+        FileNotFoundError: If any file passed as a parameter doesn't exist a FileNotFound Exception will be raised
+    """
+    for file in files:
+        if not path.isfile(file):
+            raise FileNotFoundError(f"File {file} not found")
+
+
+def ensure_columns_exists(**kwargs):
```

Comment:
this is not very readable - is kwargs really necessary? all arguments should have proper types.